src/bp/index.ts

```diff
@@ -55,11 +55,11 @@ process.PROJECT_LOCATION = process.pkg
 
 process.stderr.write = stripDeprecationWrite
 
-process.on('unhandledRejection', err => {
-  global.printErrorDefault(err)
+process.on('unhandledRejection', (reason: {} | null | undefined, promise: Promise<any>) => {
```

Comment:
Based on the documentation, it should be `Error | any`: https://nodejs.org/docs/latest-v10.x/api/process.html#process_event_unhandledrejection